slurm/SlurmProbe.py

```diff
@@ -140,7 +140,7 @@ def get_slurm_version(self):
             if os.path.exists(c):
                 cmd = c
 
-        fd = os.popen(prog)
+        fd = os.popen(cmd)
```

Comment:
This breaks the path `SlurmLocation` configuration.  `cmd` is just the `srun --version`, and `prog` should be the resolved version after accounting for `SlurmLocation`

Reply:
Thanks @djw8605!

I totally missed the second problem, that the `SlurmLocation` lookup would never succeed, since it was testing for the existence of `prog` which was set to `srun --version`.

Let me know if this fixes it.